releaseherald/docs/version_template.md

```diff
@@ -0,0 +1,4 @@
+# Version template
+
+!!! warning "This is not yet written" 
```

Comment:
Eh, is this expected?

Reply:
I would like the placeholder page there, as todo item, the best to write when I will have metadata extraction as the structure the templates works on will be much closer to final by that. Putting it into a warning box make it stands out showing that we are working on this.